app/services/form1010cg/service.rb

```diff
@@ -9,7 +9,22 @@ class InvalidVeteranStatus < StandardError
     attr_accessor :claim, # SavedClaim::CaregiversAssistanceClaim
                   :submission # Form1010cg::Submission
 
-    NOT_FOUND = 'NOT_FOUND'
+    STATSD_KEY_PREFIX = 'api.form1010cg'
+    NOT_FOUND         = 'NOT_FOUND'
+
+    def self.metrics
+      prefix = STATSD_KEY_PREFIX + '.submission'
```

Comment:
Since submission is always appended, does it make sense to move it up to the STATSD_KEY_PREFIX constant or are you planning on adding additional metrics in the future e.g. api.form1010cg.something_else?

Reply:
There may be a status check in the future, but I can move it to STATSD_KEY_PREFIX and extract it when/if there are more metric types.